database/seeders/DatabaseSeeder.php

```diff
@@ -12,11 +12,6 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        // \App\Models\User::factory(10)->create();
-
-        // \App\Models\User::factory()->create([
-        //     'name' => 'Test User',
-        //     'email' => 'test@example.com',
-        // ]);
+        \App\Models\Task::factory(10)->create();
```

Comment:
try having 10k here

Reply:
oh for 10k it will probably take a while to load. We could add chunks here if we want more data - so we get a better performance